fe-ionic/init.sh

```diff
@@ -18,11 +18,58 @@ while [[ "$#" > 0 ]]; do case $1 in
 esac; shift; done
 
 echo "generate project"
+# create temp-dir since non-interactive ionic-cli can not generate into existing directories
 mkdir start_$COMPONENT
 cd start_$COMPONENT
-ionic start $COMPONENT blank --type=angular --no-deps --no-git
+
+# creating project
+ionic start $COMPONENT blank --type=angular --no-deps --no-git --no-interactive
+
+# move generated project to the intended directory
 mv $COMPONENT/ ..
+# switch to component directory
 cd ../$COMPONENT
+# remove empty temp-dir
+rm ../start_$COMPONENT
+
+echo "Change test setup to single run in karma.conf.js"
```

Comment:
``fe-ionic/files/package.json`` was removed. On the bottom of this ``init.sh`` is still a ``rm ./package.json``. After this there wouldn't be any package.json left...

beside that the ``fe-ionic/files/package.json`` had some finetuning on the tasks like added parameters for generating code coverage analysis data:
```
"test": "ng test --code-coverage --reporters=junit",
```

Reply:
Done as requested